.github/ci_commit_pins/torchbench.txt

```diff
@@ -1 +1 @@
-766a5e3a189384659fd35a68c3b17b88c761aaac
+torchbench-pin-commit
```

Comment:
From https://github.com/pytorch/benchmark/pull/2584, to be switched to main commit once https://github.com/pytorch/benchmark/pull/2584 lands